types/office-js-preview/index.d.ts

```diff
@@ -2500,7 +1857,15 @@ declare namespace Office {
          * Returns the data contained within the binding.
          *
          * @remarks
-         * <table><tr><td>Requirement Sets</td><td>MatrixBindings, TableBindings, TextBindings</td></tr></table>
+         * **Requirement Sets:** 
```

Comment:
Does this need all the coercion items?

Reply:
They are called out in the SoT conceptual page for this method.  Again, this is just my logic. I'm open to a better system.

Reply:
ok